docker/Dockerfile

```diff
@@ -71,6 +81,43 @@ RUN mkdir /opt/aarch64-toolchain && curl $AARCH64_TOOLCHAIN_LINK | tar xJ -C /op
 # Add generic non-root user
 RUN addgroup bao && adduser -disabled-password --ingroup bao bao
 
+# Add qemu-riscv64-virt
+RUN mkdir /opt/qemu-riscv64-virt && \
+    git clone https://github.com/qemu/qemu.git /opt/qemu-riscv64-virt --depth 1 --branch v7.2.0 && \
+    cd /opt/qemu-riscv64-virt/ && ./configure --target-list=riscv64-softmmu --enable-slirp && \
+    make -j$(nproc) && \
+    make install
+
+# Add OpenSBI
+RUN mkdir /opt/OpenSBI && git clone https://github.com/bao-project/opensbi.git /opt/OpenSBI --depth 1 --branch bao/demo
```

Comment:
Why building opensbi? I don't get the purpose...

Reply:
Removed openSBI building, thanks!